SU2_CFD/src/iteration/CDiscAdjFEAIteration.cpp

```diff
@@ -196,10 +175,10 @@ void CDiscAdjFEAIteration::RegisterInput(CSolver***** solver, CGeometry**** geom
 
     solver[iZone][iInst][MESH_0][FEA_SOL]->RegisterVariables(geometry[iZone][iInst][MESH_0], config[iZone]);
 
-    /*--- Register mesh coordinates for geometric sensitivities ---*/
-
-    geometry[iZone][iInst][MESH_0]->RegisterCoordinates(config[iZone]);
   }
+  /*--- Register mesh coordinates for geometric sensitivities ---*/
+
+  geometry[iZone][iInst][MESH_0]->RegisterCoordinates(config[iZone]);
```

Comment:
I think I found a better fix for this in #1284. (better because always registering the coordinates increases memory usage)
We can leave it like this in this PR though.